salt/_states/metalk8s_kubernetes.py

```diff
@@ -129,6 +129,9 @@ def object_present(name, manifest=None, **kwargs):
 
     manifest_metadata = (manifest_content or {}).get("metadata", {})
 
+    if manifest_metadata is None:
+        manifest_metadata = {}
```

Comment:
Not sure it's needed, we are in this situation here just because in the manifest we have `metadata:` (without anything in it so `None`)